### ISSUE TITLE
write_h5ms lost index 

### ISSUE THREAD

**Issue by LILI-0000-0002-8173-7367** [STOmics/Stereopy#270]
Hello, Stereopy team.

I noticed that some attributes were lost once ms_data object was saved as h5ms file formart, related to post #234. 

originial ms_data
![image](https://github.com/STOmics/Stereopy/assets/137732527/5c40b73d-56ca-48d2-99e1-d8278bd1b035)
![image](https://github.com/STOmics/Stereopy/assets/137732527/c631f57f-6a5d-48c7-9ca6-fe6bb64d8049)
![image](https://github.com/STOmics/Stereopy/assets/137732527/1e72c31a-8979-4ef4-a1a3-467fa7801914)

reloaded ms_data1 from h5ms file.
![image](https://github.com/STOmics/Stereopy/assets/137732527/9154dd1f-f86a-4bd2-ac67-e26ad9adbf9d)
![image](https://github.com/STOmics/Stereopy/assets/137732527/14b58d37-e38c-45de-a464-76eb9ec51457)
![image](https://github.com/STOmics/Stereopy/assets/137732527/b8a50ada-1bff-421b-ab0e-4d2e438b316d)

You will noticed that leiden show up in ms_data1.obs and mean_umi is completely disappreared in ms_data.var.
Basically I can replot anything from this h5ms file. Would you please help?
``
st.__version__ 
'1.2.0'
``
``
st.io.write_h5ms(ms_data,output='/work/InternalMedicine/sssss/Stereopy/bin50_ALL.h5ms')
``
``
ms_data1 = st.io.read_h5ms(file_path ='/work/InternalMedicine/sssss/Stereopy/bin50_ALL.h5ms',
                use_raw=True, use_result=True)
``

**Comment by tanliwei-genomics-cn** [comment]
We have already solved this problem in latest version stereopy, thanks so much for your feedback.
